tests/ttnn/unit_tests/operations/eltwise/test_binary_fp32.py

```diff
@@ -551,3 +551,47 @@ def test_bitwise_xor(device, ttnn_function):
 
     status = ttnn.pearson_correlation_coefficient(z_torch, tt_out) >= 0.999
     assert status
+
+
+@skip_for_grayskull("Unsupported dtype for Grayskull")
+@pytest.mark.parametrize(
+    "ttnn_function",
+    [
+        ttnn.bitwise_left_shift,
+    ],
+)
+def test_bitwise_left_shift(device, ttnn_function):
+    x_torch = torch.tensor([[99, 3, 100, 1, 72, 0]], dtype=torch.int32)
+    y_torch = torch.tensor([[1, 2, 31, 4, 5, 0]], dtype=torch.int32)
+    golden_fn = ttnn.get_golden_function(ttnn_function)
+    z_torch = golden_fn(x_torch, y_torch)
+    x_tt = ttnn.from_torch(x_torch, dtype=ttnn.int32, layout=ttnn.TILE_LAYOUT, device=device)
+    y_tt = ttnn.from_torch(y_torch, dtype=ttnn.int32, layout=ttnn.TILE_LAYOUT, device=device)
+    z_tt = ttnn.from_torch(z_torch, dtype=ttnn.int32, layout=ttnn.TILE_LAYOUT, device=device)
+    z_tt_out = ttnn.bitwise_left_shift(x_tt, y_tt)
+    tt_out = ttnn.to_torch(z_tt_out)
+
+    status = ttnn.pearson_correlation_coefficient(z_torch, tt_out) >= 0.999
+    assert status
+
+
+@skip_for_grayskull("Unsupported dtype for Grayskull")
+@pytest.mark.parametrize(
+    "ttnn_function",
+    [
+        ttnn.bitwise_right_shift,
+    ],
+)
+def test_bitwise_right_shift(device, ttnn_function):
+    x_torch = torch.tensor([[19, 3, 101, 21, 47, 0]], dtype=torch.int32)
+    y_torch = torch.tensor([[5, 2, 31, 4, 5, 0]], dtype=torch.int32)
```

Comment:
for right_shift, "y" is limited 0<=y<32, but "x" can be a negative integer which is a corner case that I implemented into the LLK. It is a corner case because 1's have to be shifted into the output value as opposed to 0's, so please add one more case which is a negative x value.

Reply:
sure will add these cases and update